lib/util/date.ts

```diff
@@ -10,3 +12,7 @@ export function getElapsedDays(timestamp: string): number {
 export function getElapsedMinutes(date: Date): number {
   return Math.floor((new Date().getTime() - date.getTime()) / ONE_MINUTE_MS);
 }
+
+export function fixShortHours(input: string): string {
+  return input.replace(regEx(/( \d?\d)((a|p)m)/g), '$1:00$2');
```

Comment:
@viceice I'm not sure about this relocation.. it's very later-specific, and I think belongs in migration. Or do you think there's a better reason for it being here?

Reply:
Yeah, let's move to migration code. Hopefully we can remove the call from worker code soon

Reply:
@RahulGautamSingh could you make this change?

Reply:
@rarkins Done.